Source/WTF/Scripts/Preferences/WebPreferencesExperimental.yaml

```diff
@@ -405,6 +405,18 @@ CSSRelativeColorSyntaxEnabled:
     WebCore:
       default: false
 
+CSSScrollAnchoringEnabled:
+  type: bool
+  humanReadableName: "CSS Scroll Anchoring"
+  humanReadableDescription: "Enable CSS Scroll Anchoring support"
```

Comment:
The word "support" doesn't add anything.

Reply:
Fixed.